.github/scripts/end2end/configs/zenkoversion.yaml

```diff
@@ -58,7 +58,7 @@ spec:
       image: '${CLOUDSERVER_IMAGE}' 
       tag: '${CLOUDSERVER_TAG}'
     backbeat:
-      image: '${BACKBEAT_IMAGE}' 
+      image: '${BACKBEAT_IMAGE}'
```

Comment:
no need to change (and should not even be in history, it uses introduces noise after the PR is merged)